### ISSUE TITLE
Error reporting

### ISSUE THREAD

**Issue by ataiya** [textmate/latex.tmbundle#152]
Very often something goes wrong and all I get is this:

```
Could not find any messages containing line information.
Please take a look at the log file engage.latexmk.log to find the source of the problem.
```

It would be nice if in these situations the LatexWatch could just dump the log file directly in the window, no?

**Comment by sorbits** [comment]
As an intermediate step, the text could be made into a link (if this is not already the case).

Here’s a simple example of using JavaScript to open a file (from TextMate’s HTML output):

```HTML
Open <tt><a href="#" onClick="javascript:TextMate.open('/etc/hosts')">/etc/hosts</a></tt>
```

**Comment by sanssecours** [comment]
> It would be nice if in these situations the LatexWatch could just dump the log file directly in the window, no?

Do you mean that we should:

1. open the file `filename.latexmk.log` in a new tab in TextMate, or
2. display the content of the logfile in the Notification Window.

I do not think that either of this options is ideal. Especially option 2 seems kind of extreme. I think it would be better to just to improve the parser code to catch more helpful error messages. Hopefully I will have time to update the parser code in the next weeks.

> As an intermediate step, the text could be made into a link (if this is not already the case).

Thanks, for the tip Allan. But I think in the situation Andrea described only the [Notification Window](https://github.com/textmate/latex.tmbundle/issues/73#issuecomment-67341005) is open. As far as I know there is no (easy way) to add a link there…

**Comment by ataiya** [comment]
I agree it's not ideal, but better than having to chase down the log file by hand.

A link would've been optimal!
Even better if errors where better caught but there is always room for things to go wrong!
